src/api/app/views/webui2/shared/_buildresult_box.html.haml

```diff
@@ -0,0 +1,87 @@
+- index ||= ''
```

Comment:
I'm not sure we should carry over the `shared` pattern. I know these partials are shared between `Package` & `Project` but I always found this confusing. What do you think?